ftplugin/latex-suite/outline.py

```diff
@@ -100,9 +105,11 @@ def getSectionLabels_Root(lineinfo, section_prefix, label_prefix):
             # :          e^{i\pi} + 1 = 0
             #
             # Use the current "section depth" for the leading indentation.
-            print >>outstr, '>%s%s\t\t<%s>' % (' ' * (2 * pres_depth + 2),
+            print('>%s%s\t\t<%s>' % (' ' * (2 * pres_depth + 2),
```

Comment:
I'm not familiar with that function. I'm sure I can change it. I just wanted to stay as faithful as possible to the pre-existing code.